### ISSUE TITLE
Alleviate concerns about small group making big decisions.

### ISSUE THREAD

**Issue by e-lo** [MetropolitanTransportationCommission/travel-modeling-as-science#7]
- at the end of the day you still have the same basic problem of a small number of people making important decisions for the community. How do alleviate these concerns?

**Comment by e-lo** [comment]
@DavidOry  you are right, but I think there are a few advantages here:
 a - it is explicit who is making the decision [ this was the case in original strawman as well ] as opposed to happening at random meetings between people in the halls of TRB  or by an AASHTO committee
b - this program is opt-in by membership, so supposedly you would only opt-in if you thought they were doing a good job  

That said...
c - I think there is still some thinking to do regarding how to get a good balance of stakeholder _types_  on the board while still being responsive to _member_ desires.  @billyc  and I have some ideas about "types of people" who would make a good first board that we can discuss with you.

**Comment by DavidOry** [comment]
I should say that I do not share this concern.  We need people to make decisions.  And all public entities and non-profits are, to a great extent, reflections of their leadership.  I would advocate for a means for the community to efficiently get rid of people who are making bad decisions (one easy way to do this is to rotate people off after a fixed tenure).

**Comment by billyc** [comment]
@DavidOry I agree every organization is a reflection of its leadership; that can't be avoided. In our revision, we propose carefully selecting the initial Board, and having some sort of community-led process after the initial term (an election is mentioned, but an election may not be ideal; we should think about other methods as well).

**Comment by gregerhardt** [comment]
+1 to Billy and Dave.  I think you craft it in whatever vision you have to get things off the ground, and then worry about opening it up more broadly.  If you try to meet every possible need and stakeholder desire early on, it can easily get stuck in the mud.
